modules/Microsoft.Authorization/policySetDefinitions/.test/mg/deploy.test.bicep

```diff
@@ -0,0 +1,64 @@
+targetScope = 'managementGroup'
```

Comment:
Please change the folder name to `mg.common`

Reply:
done